CONTRIBUTING.md

```diff
@@ -106,24 +114,32 @@ $ yarn -s compress
 $ yarn size-min-gz
 ```
 
-**Note**: If the min+gz size increases, please note it in the README. If it is a significant increase, please flag to your reviewers and have a discussion about whether or not the size addition is justified.
+**Note**: If the min+gz size increases, please note it in the README. If it is a significant increase,
+please flag to your reviewers and have a discussion about whether or not the size addition is justified.
 
 ## Before submitting a PR...
 
-Before you go ahead and submit a PR, make sure that you have done the following:
+... please make sure that you have done the following:
 
-```sh
-$ yarn run test
-$ yarn run benchmark
-```
+1. Confirm that all checks are passing:
+
+   ```sh
+   $ yarn run test
+   $ yarn run benchmark
+   ```
+
+2. Confirm we don't have any significant performance regressions (check out `master` for a baseline comparison on _your_ machine).
+
+3. Confirm you aren't impacting our benchmark stats.
```

Comment:
This is really “bundle size” not “benchmark stats” (which instead deals with how many operations versus other libraries for performance). 

I think what we really want is a separated section for always checking the bundle size and updating the documents accordingly

Reply:
We do have the Size section of this document, but it only outlines the steps necessary to check on that size, and not what to update in turn. Should I expand on the section in Size and keep the directions here? Or break out a different section entirely?